beanstalk_worker/worker_start_scan.py

```diff
@@ -110,7 +110,8 @@ def run(self):
 
         # TODO: Figure out why random tag (with date) is coming
         # image_under_test = ":".join(self.job_info.get("name").split(":")[:-1])
-        image_under_test = self.job_info.get("name")
+        image_under_test = self.job_info.get("name") + ":" \
+            + self.job_info.get("tag")
```

Comment:
Yep, I did that just before making earlier comment. And I'm getting proper mail notifications for at least one image that I built which doesn't use `latest` tag.